packages/plugin-session-replay/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@amplitude/plugin-session-replay",
```

i believe our package naming convention is including the platform at the end cc @justin-fiedler 

```suggestion
  "name": "@amplitude/plugin-session-replay-browser",
```